### ISSUE TITLE
index.js npm

### ISSUE THREAD

**Issue by kyoukhana** [gerardreches/vue-qrcode-component#2]
I find when compiling with webpack. I needed to change the following.

From
```
module.exports = QRCode
```

To
```
import QRCode from './src/QRCode.vue'

export default QRCode
```

**Comment by gerardreches** [comment]
I will take a look at this, thanks for reporting.

**Comment by gerardreches** [comment]
Fixed in [v2.1.0](https://github.com/gerardreches/vue-qrcode-component/releases/tag/v2.1.0)
